client-app/src/desktop/tabs/inputs/tabs/TimeInputPanel.js

```diff
@@ -0,0 +1,70 @@
+import {useLocalModel} from '@xh/hoist/core/hooks';
+import {InputTestModel} from '../InputTestModel';
+import {PropTypes as T} from 'react-view';
+import {TimeInput} from '@xh/hoist/desktop/cmp/input';
+import {Icon} from '@xh/hoist/icon';
+import {hoistCmp} from '@xh/hoist/core';
+import {inputTestPanel} from '../InputTestPanel';
+
+export const TimeInputPanel = hoistCmp({
```

Comment:
This file does not look like it should be included in this pull request. Was this branch created directly off of develop?